src/components/Profile.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+import user from '../data/user';
+function Profile({ currentUser, setCurrentUser }) {
+  const handleChangeUser = () => {
+    currentUser ? setCurrentUser(0) : setCurrentUser(1);
+  };
+  return (
+    <ProfileWrapper onClick={handleChangeUser}>
+      <ProfileImg
+        src={process.env.PUBLIC_URL + '/img/' + user[currentUser].profileImg}
+      />
+      <NameWrapper>
+        <Name>{user[currentUser].name}</Name>
+        <Status>{user[currentUser].status}</Status>
+      </NameWrapper>
+    </ProfileWrapper>
```

Comment:
```js
const { profileImg, name, status } = user[currentUser]
```

```suggestion
  return (
    <ProfileWrapper onClick={handleChangeUser}>
      <ProfileImg
        src={process.env.PUBLIC_URL + '/img/' + profileImg}
      />
      <NameWrapper>
        <Name>{name}</Name>
        <Status>{status}</Status>
      </NameWrapper>
    </ProfileWrapper>
```

object의 attribute를 계속해서 사용한다면 destructing 해주는게 가독성에 좋죠!